metasmoke.py

```diff
@@ -57,6 +58,17 @@ def init_websocket():
                     time.sleep(10)
 
     @staticmethod
+    def check_last_pingtime():
+        while True:
```

Comment:
Instead of looping forever and ever, you might consider only running this check once every few seconds and making a new `threading.Timer`, like we do in `send_status_ping`.